source/sdk/math/Random.ooc

```diff
@@ -39,6 +40,17 @@ Random: class {
 	randFloat: static func -> Float {
 		return (rand() as Float) / INT_MAX
 	}
+	/**
+	   Box-Muller transform, http://en.wikipedia.org/wiki/Box-Muller_transform
+	   :return: a pseudo-random float number with a normal distribution
```

Comment:
`:return: a pseudo-random float number with a normal distribution, centered about mu`. for clarity.